src/Commands/MigrateFresh.php

```diff
@@ -19,7 +19,10 @@ class MigrateFresh extends Command
      *
      * @var string
      */
-    protected $signature = 'migrate:fresh {--seed} {--force}';
+    protected $signature = 'migrate:fresh {--database= : The database connection to use.}
+                {--force : Force the operation to run when in production.}
+                {--path= : The path of migrations files to be executed.}
```

Comment:
This is a copy/past of https://github.com/laravel/framework/blob/5.4/src/Illuminate/Database/Console/Migrations/MigrateCommand.php